rfc6962-bis.xml

```diff
@@ -401,6 +401,22 @@ REDACT(label) = prefix || BASE32(index || LABELHASH(keyid || label))</artwork>
             This extension informs TLS clients of the dNSNames that were redacted and the degree of redaction, while minimizing the complexity of TBSCertificate reconstruction (as described in <xref target="reconstructing_tbscertificate"/>). Hashing the redacted labels allows the legitimate domain owner to identify whether or not each redacted label correlates to a label they know of.
           </t>
         </section>
+        <section title="Ensuring Effective Redaction">
+          <t>
+            Although this redaction mechanism removes the need for private labels to appear in logs, it does not guarantee that this will never happen. Anyone who encounters a certificate could choose to submit it to one or more logs, thereby rendering the redaction futile. Therefore, it is RECOMMENDED that domain owners take the following steps to minimize the likelihood that their private labels will become known outside their closed communities:
+            <list style="symbols">
+              <t>
+                Avoid registering private labels in public DNS.
+              </t>
+              <t>
+                Avoid using private labels that are predictable (e.g. <spanx style="verb">www</spanx>).
+              </t>
+            </list>
+          </t>
+          <t>
+            CAs SHOULD carefully consider each request to redact a label. When a CA believes that redacting a particular label would be futile, the CA SHOULD NOT redact it. TLS clients may have policies that forbid redaction, so redaction should only be used when it's absolutely necessary and likely to be effective.
```

Comment:
s/SHOULD/should/g, as it's not a part of the protocol, but a recommendation for CAs.